doc/Command-Reference.md

```diff
@@ -3786,6 +3786,172 @@ This command is used to delete a configured NTP server IP address.
 Go Back To [Beginning of the document](#) or [Beginning of this section](#NTP)
 
 
+## Firmware
```

Comment:
I think maybe we can rename this section to something like "Component Firmware", because "Firmware" is vague and might be misleading to some users, because SONiC itself is sometimes referred to as firmware.

Also, once a new name is decided upon, please move this new section so that it is in alphabetical order with respect to the names of other sections, and also add links in the Table of Contents.

Reply:
@jleveque fixed